### PR TITLE
Using dynamic environment

### PR DESCRIPTION
**Pull request by trunges21**
# Explain your changes

Addresses #14.

# Checklist

- [x] I have read the [“Pull requests” section](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md#pull-requests) in the [contributing guidelines](https://github.com/kinde-oss/.github/blob/main/.github/CONTRIBUTING.md).
- [x] I agree to the terms within the [code of conduct](https://github.com/kinde-oss/.github/blob/main/.github/CODE_OF_CONDUCT.md).

🛟 _If you need help, consider asking for advice over in the [Kinde community](https://thekindecommunity.slack.com)._

### CODE REVIEW

**Comment by oliwolff1** [comment]
This PR is fixing a issue where env variables are imported only at builtime.

**Comment by atifcppprogrammer** [comment]
This LGTM :+1: , I created an issue against this PR linking to the original comment made in slack for more context.

**Review by atifcppprogrammer — APPROVED** [review]

File: src/lib/config/config.ts [hunk 1]
```diff
@@ -1,25 +1,15 @@
...
-	authUsePKCE: [true, 'true'].includes(KINDE_AUTH_WITH_PKCE)
+	authDomain: env.KINDE_ISSUER_URL,
+	clientId: env.KINDE_CLIENT_ID,
+	logoutRedirectURL: env.KINDE_POST_LOGOUT_REDIRECT_URL,
+	redirectURL: env.KINDE_REDIRECT_URL,
+	audience: env.KINDE_AUDIENCE,
+	scope: env.KINDE_SCOPE,
+	clientSecret: env.KINDE_CLIENT_SECRET,
+	loginRedirectURL: env.KINDE_POST_LOGIN_REDIRECT_URL,
+	authUsePKCE: [true, 'true'].includes(env.KINDE_AUTH_WITH_PKCE)
```
Comment (atifcppprogrammer) [inline src/lib/config/config.ts:14]: Small nitpick this, but I don't think you need explicitly check for `true` here. I think `authUsePKCE: env.KINDE_AUTH_WITH_PKCE === 'true'` would perhaps make more sense ?

Comment (oliwolff1) [reply src/lib/config/config.ts:14]: Thanks for this feedback @atifcppprogrammer.
I will pass this feedback on.

**Pull request merged**